util/kvencoder/allocator.go

```diff
@@ -19,38 +19,59 @@ import (
 	"github.com/pingcap/tidb/meta/autoid"
 )
 
-var _ autoid.Allocator = &allocator{}
+var _ autoid.Allocator = &Allocator{}
 
 var (
 	step = int64(5000)
 )
 
-// NewAllocator new an allocator.
+// NewAllocator new an Allocator.
 func NewAllocator() autoid.Allocator {
-	return &allocator{}
+	return &Allocator{}
 }
 
-type allocator struct {
+// Allocator is a id allocator, it is only used in lightning.
+type Allocator struct {
 	base int64
 }
 
-func (alloc *allocator) Alloc(tableID int64) (int64, error) {
+// Alloc allocs the next autoID for table with tableID.
+func (alloc *Allocator) Alloc(tableID int64) (int64, error) {
 	return atomic.AddInt64(&alloc.base, 1), nil
 }
 
-func (alloc *allocator) Rebase(tableID, newBase int64, allocIDs bool) error {
+// ResetBase allow newBase smaller than alloc.base, and will set the alloc.base to newBase.
+func (alloc *Allocator) ResetBase(newBase int64) {
```

Comment:
why do you want to change the function name?

Reply:
the func is added, not changed.

Reply:
I think "Reset" is clear enough.